config/initializers/cors.rb

```
@@ -5,12 +5,21 @@
+#    allow do
+#      origins "*"
 #
-#     resource "*",
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+#      resource "*",
+#        headers: :any,
+#        methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#    end
```

rubocop (1.69.1)

[convention] Incorrect indentation detected (column 0 instead of 4).

(Layout/CommentIndentation)